plugins/inputs/webhooks/papertrail/README.md

```diff
@@ -0,0 +1,33 @@
+# papertrail webhooks
+
+Enables Telegraf to act as a [Papertrail Webhook](http://help.papertrailapp.com/kb/how-it-works/web-hooks/).
+
+## Events
+
+See [here](http://help.papertrailapp.com/kb/how-it-works/web-hooks/#callback) for full documentation.
```

Comment:
I prefer links without name `here`. You could make the full sentence clickable, such as `[See the full documentation](http://help.papertrailapp.com/kb/how-it-works/web-hooks/#callback) `.